packages/extension-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@looker/extension-utils",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I think we want to let release-please bump this

Reply:
done